apps/onboarding/.env.local.example

```diff
@@ -3,7 +3,13 @@ RELAYER_HOST=localhost
 RELAYER_PORT=5050
 DB_HOST=localhost
 DB_PORT=5432
-DB_USERNAME='postgres'
```

Comment:
maybe need to revert this?

Reply:
Yeah we can leave the defaults in the example files as they were before. 
What I sent in slack was for my local config, and it was more in order to give you the rest of the configs.